File: lavanderia.pro/api/types/delivery.go
package types

import "time"

type Delivery struct {
	ID        string     `json:"id" bson:"_id,omitempty" uri:"id"`
	Name      string     `json:"name"`
	Auth      string     `json:"auth,omitempty" bson:"auth,omitempty"`
	CreatedAt *time.Time `json:"created_at,omitempty" bson:"created_at"`
	UpdatedAt *time.Time `json:"updated_at,omitempty" bson:"updated_at"`
	DeletedAt *time.Time `json:"deleted_at,omitempty" bson:"deleted_at"`
}

File: app/lib/models/delivery.dart
class Delivery {
  final String? id;
  final String? name;
  final String? createdAt;
  final String? updatedAt;

  Delivery({this.id, this.name, this.createdAt, this.updatedAt});

  factory Delivery.fromJson(Map<String, dynamic> json) {
    return Delivery(
      id: json['id'],
      name: json['name'],
      createdAt: json['created_at'],
      updatedAt: json['updated_at'],
    );
  }
}